source/sliding_puzzle.py

```diff
@@ -112,7 +127,7 @@ def solved(self) -> bool:
         return True
 
     def _join(self, row: List[PuzzlePiece]) -> List[str]:
-        """Join together a row of puzzle pieces into a complete list of lines."""
+        """Join together a row of puzzle rows into a complete list of lines."""
         res = []
```

Comment:
Should probably rewrite this doc-string completely, "a row of puzzle rows" isn't quite right haha

Reply:
yeah sorry for that, I didn't pay attention, I just used the autoselect tu replace all ```pieces```, I will fix that